sqlserver/assets/configuration/spec.yaml

```diff
@@ -713,6 +713,28 @@ files:
         type: number
         example: 1800
         display_default: false
+    - name: schemas_collection 
+      description: |
+        Configure collection of schemas.
+      options:
+        - name: enabled
+          description: |
+            Enable schema collection. Requires `dbm: true`. Defaults to true.
+          value:
+            type: boolean
+            example: true
+        - name: collection_interval
+          description: |
+            Set the database schema collection interval (in seconds). Defaults to 600 seconds.
+          value:
+            type: number
+            example: 600
+        - name: max_execution_time
+          description: |
+            Set the maximum time for schema collection (in seconds). Defaults to 10 seconds. Capped by `collection_interval`
```

Comment:
what collection_interval are you referring to here? There are quite a few. Would be specific that this refers to the collection interval of the sqlserver integration, which IIRC is configured via `min_collection_interval` at the check level

Reply:
cc @lu-zhengda can you confirm that? ☝️